doc/mkdocs/docs/css/custom.css

```diff
@@ -0,0 +1,4 @@
+/* disable ligatures in code and preformatted blocks */
+code, pre {
+    font-variant-ligatures: none;
```

Comment:
Is this a concern for you? I am using Jetbrains Mono and enjoy their ligatures.

Reply:
Most annoyingly it turns `<=>` into iff (if and only if). The other operators are also subject to ligatures which is slightly less distracting.

Reply:
I have never perceived it as iff - but I guess now I cannot unsee it :)